docs/static/releasenotes.asciidoc

```diff
@@ -1,5 +1,128 @@
+
+*`Elasticsearch`*:
+
+* Added a pipeline configuration option for setting an ingest pipeline to run upon indexing.
+
+[[logstash-2-3-2]]
+=== Logstash 2.3.2 Release Notes
+
+* Added reload support to the init script so you can do `service logstash reload`.
+* The original intent of `"%{foo}"` syntax was to always produce a string, but the previous 2.x and 1.5.x versions
+returned a float (underlying value). Starting in release 2.3.0, Logstash returned the underlying value type for
+field references. In 2.3.2 we are returning back to the 2.x and 1.5.x behavior of returning a float because changing
+the behavior broke compatibility in a minor release ({lsissue}5114[Issue 5114]).
+* Fixed use of the `KILL_ON_STOP_TIMEOUT` variable in init scripts, which allows Logstash to force stop ({lsissue}4991[Issue 4991]).
+   
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fixed an issue encountered when the time-based flush feature was used with the multiline codec
+(https://github.com/logstash-plugins/logstash-input-beats/issues/73[Issue 73]).
+
+*`HTTP Poller`*:
+
+* Fixed the `ssl_certificate_validation` option so that it actually lets you disable cert validation 
+(https://github.com/logstash-plugins/logstash-input-http_poller/issues/48[Issue 48]).
+
+[float]
+==== Filter Plugins
+
+*`XML`*:
+
+* Added a setting called `force_array` that when set to false prevents storing single elements in arrays
+(https://github.com/logstash-plugins/logstash-filter-xml/issues/27[Issue 27]).
+
+[float]
+==== Output Plugins
+
+*`Redis`*:
+
+* Fixed a flood of runtime warnings that were logged when the Redis output was used (https://github.com/logstash-plugins/logstash-output-redis/issues/26[Issue 26]).
+
+[[logstash-2-3-1]]
+=== Logstash 2.3.1 Release Notes
+
+* Fixed a JRuby thread safety issue that was encountered when using regular expressions under multiple workers
+({lsissue}4977[Issue 4977]).
+* Disabled environment variables interpolation by default. This feature is experimental in Logstash 2.3.1. To turn it on use the `--allow-env` flag ({lsissue}4958[Issue 4958]). 
+* Changed the Logstash shutdown process to hide sensitive data from the log when shutting down a stale Logstash
+({lsissue}4952[Issue 4952]).
+* Disabled the default behavior of outputting the Configuration AST when running Logstash in debug mode. Introduced the `--debug-config` flag to display the AST ({lsissue}4965[Issue 4965]).
+* Fixed an error encountered when running Logstash with the `--config-test` flag ({lsissue}4933[Issue 4933]).
+* Made filter conditionals work when running Logstash with automatic configuration reloading ({lsissue}4968[Issue 4968]).
+* Fixed the stop command of the Ubuntu init script ({lsissue}4940[Issue 4940]).
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Changed when the identity map is used. Now it's only used when the configuration specifies the multiline codec 
+(https://github.com/logstash-plugins/logstash-input-beats/pull/70[Issue 70]).
+
+
+[[logstash-2-3]]
```

Comment:
This needs to be 2.4 release notes, no? I don't think it should include any release notes in 2.3.x series